airflow/www/webpack.config.js

```diff
@@ -35,6 +36,22 @@ const JS_DIR = path.resolve(__dirname, './static/js');
 // noinspection JSUnresolvedVariable
 const BUILD_DIR = path.resolve(__dirname, './static/dist');
 
+// Convert licenses json into a standard format for LICENSES.txt
+const formatLicenses = (packages) => {
+  let text = `Apache Airflow
+Copyright 2016-2021 The Apache Software Foundation
```

Comment:
Could we interpolate the `2021` to save us the hassle of modifying two months from now?

Reply:
I copied this from `NOTICE`. Is there a way to interpolate it there too?

Reply:
I believe it's not allowed.

My interpretation is that you have to state what was the year your contributions were made and it should not be 
  a) future, because for example it would limit you so that you could not change licencing scheme until it expires 
  b) present - because it is meaningless in the context of "writer". The `present` only makes sense in the context of "reader" because you do not keep in the file the date when "present" was put. However, the licence file needs the context of "writer", so you need to specify when was the time the licence was last "expected" to be applied by those who have the rights to provide the licence.

We usually perform a bulk update of all licence dates at the beginning of the year. This is also clear expression of intent "yes we want to licence our code under the ASF licence also in 2022". So do not worry about it now.

We usually update all the (C) notices

Reply:
I believe it's not allowed.

My interpretation is that you have to state what was the year your contributions were made and it should not be 
  a) future, because for example it would limit you so that you could not change licencing scheme until it expires 
  b) present - because it is meaningless in the context of "writer". The `present` only makes sense in the context of "reader" because you do not keep in the file the date when "present" was put. However, the licence file needs the context of "writer", so you need to specify when was the time the licence was last "expected" to be applied by those who have the rights to provide the licence.

We usually perform a bulk update of all licence dates at the beginning of the year. This is also clear expression of intent "yes we want to licence our code under the ASF licence also in 2022". So do not worry about it now.

We usually update all the (C) notices